### ISSUE TITLE
index.md contents not showing at all

### ISSUE THREAD

**Issue by gelavat** [BesrourMS/clean-blog#14]
I have installed clean-blog out of the box, latest version to date, with latest Pico CMS to date.

The logo is properly displayed, and the text in the "Tagline:" in content/index.md file.
However everything I put below the meta header is not shown at all.

Can you please help? Also, where can I find documentation on how to use clean-blog theme please (add contents, add menus, etc...)?

**Comment by mayamcdougall** [comment]
I think that's the point of this theme.  It's a blog-style theme. The front page is *supposed* to be ignored, in favor of using the index page to display blog style headlines from each of the other pages.

It also says it depends on the [Pico-Pagination](https://github.com/rewdy/Pico-Pagination) plugin.  I don't see any backup code for if you're not using Pagination, so if it just shows a blank page under the header, you'll probably need that plugin to get the front page working.

**Comment by gelavat** [comment]
Thank you. Yes that is right, the contents of the index.md is not shown, probably in purpose.
And the issue was that I did not install the Pico-Pagination plugin. It was only told that it was used, but unclear if used to create the theme or mandatory for the theme to work nor if it was automatically installed with the theme. Again we don't know the details when we come as first comers.

I corrected all this and provided a PR that has been accepted. Now all works fine and can be installed and customized properly I think.
